common/network-shuffle/src/main/java/org/apache/spark/network/shuffle/RetryingBlockTransferor.java

```diff
@@ -26,6 +27,7 @@
 
 import com.google.common.collect.Sets;
 import com.google.common.util.concurrent.Uninterruptibles;
+import org.apache.spark.network.sasl.SaslTimeoutException;
```

Comment:
`org.apache` imports come last.
See [here](https://github.com/databricks/scala-style-guide#imports) for details.

Reply:
done